data/fields/vhf.json

```diff
@@ -1,6 +1,6 @@
 {
     "key": "vhf",
-    "type": "text",
```

Comment:
this change is unrelated to the wilderness hut preset, isn't it?

I'm wondering if this was made a text field on purpose because of the "special" usage as double digit numbers [wiki](https://wiki.openstreetmap.org/wiki/Key%3Avhf):

> Some people prefer for the channel numbers below 10 a leading zero (0), for instance "09".

Reply:
(ping?)

Reply:
sorry, that change wasn't meant to be part of this PR. I tried to remove that change from a different branch and obviously messed something up...